build.sbt

```diff
@@ -203,6 +207,22 @@ lazy val monix = crossProject(JSPlatform, JVMPlatform)
 lazy val monixJVM = monix.jvm
 lazy val monixJS  = monix.js
 
+lazy val monixBio = crossProject(JSPlatform, JVMPlatform)
+  .crossType(CrossType.Pure)
+  .in(file("modules/monixBio"))
+  .dependsOn(core, framework % "test->compile")
+  .configure(WeaverPlugin.profile)
+  .settings(WeaverPlugin.simpleLayout)
+  .settings(
+    libraryDependencies ++= Seq(
+      "io.monix" %% "monix-bio" % "1.0.0"
+    ),
+    scalaJSLinkerConfig ~= { _.withModuleKind(ModuleKind.CommonJSModule) }
```

Comment:
I'm hoping we can merge #82 before this, so that we don't have to emit commonjs modules exclusively anymore.